src/lib/db.ts

```diff
@@ -32,6 +32,7 @@ import Config from '../shared/config'
 import ConsentDB from '../model/consent'
 import ScopeDB from '../model/scope'
 
+// eslint-disable-next-line @typescript-eslint/ban-types
 const Db: Knex = Knex(Config.DATABASE as object)
```

Comment:
here shouldn't typecasting be needed - if it is then Config.DATABASE is incompatible with Knex constructor signature

Reply:
these are all existing linting issues, will fix it in separate story.